generate_docs.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+ORIG_DIR=$(echo $PWD)
+PKG=$(echo $1)
+cd ${PKG}/integration_tests
+dbt clean && dbt deps && dbt seed && dbt run
```

Comment:
Do we need to run `dbt run`? Could we limit the steps by just executing `dbt clean && dbt deps && dbt seed && dbt docs generate`?